scripts/install/RegisterTestRunnerFeatureService.php

```diff
@@ -21,25 +21,26 @@
 
 namespace oat\taoTests\scripts\install;
 
-use oat\oatbox\service\ServiceManager;
+use oat\oatbox\extension\InstallAction;
 use oat\taoTests\models\runner\features\TestRunnerFeatureService;
 
 /**
  * Installation action that registers the TestPluginService
  *
  * @author Christophe Noël <christophe@taotesting.com>
  */
-class RegisterTestRunnerFeatureService extends \common_ext_action_InstallAction
+class RegisterTestRunnerFeatureService extends InstallAction
 {
     /**
      * @param $params
+     * @throws \common_Exception
+     * @throws \common_exception_Error
      */
     public function __invoke($params)
     {
-        $serviceManager = ServiceManager::getServiceManager();
-
+        $serviceManager = $this->getServiceManager();
         $testRunnerFeatureService = new TestRunnerFeatureService();
-        $testRunnerFeatureService->setServiceManager($serviceManager);
+        $serviceManager->propagate($testRunnerFeatureService);
```

Comment:
the propagate is already included in the register, you don't need to call it explicitly